### ISSUE TITLE
editor breaks production builds

### ISSUE THREAD

**Issue by cluetjen** [telerik/kendo-angular#2200]
When using `"@progress/kendo-angular-editor": "^0.4.0",` angular cli production builds will fail due to wrong parameter count:

```
ERROR in node_modules\@progress\kendo-angular-editor\dist\es2015\index.d.ts.FileLinkDialogComponent.html(10,64): : Expected 0 arguments, but got 1.
node_modules\@progress\kendo-angular-editor\dist\es2015\index.d.ts.FileLinkDialogComponent.html(17,64): : Expected 0 arguments, but got 1.
node_modules\@progress\kendo-angular-editor\dist\es2015\index.d.ts.FileLinkDialogComponent.html(24,65): : Expected 0 arguments, but got 1.
```

**To Reproduce**

Run `ng build --prod`

**Browser**
Command line

**Additional context**

In file-link-dialog.component:

Template

     <label (click)='hrefInput.focus()'>Web address{{ textFor('fileWebAddress') }}</label>

Component

     textFor() {
        //TOOD localize
        //return this.localization.get(key);
        return '';
     }

Explains "\index.d.ts.FileLinkDialogComponent.html(17,64): : Expected 0 arguments, but got 1."

**Comment by Raisolution** [comment]
Hi @cluetjen,

Thank you for reporting the issue. This code is part of the upcoming Editor localization and slipped incidentally in production. A fix has been released in `v4.1.0`. Let us know if everything is OK with it.

**Comment by cluetjen** [comment]
Hi @Raisolution, thanks, that was fast ;-) Yes, works again.
